logfire/_internal/config.py

```diff
@@ -679,6 +692,8 @@ def _initialize(self) -> None:
                 )
             if self.service_version:
                 otel_resource_attributes[ResourceAttributes.SERVICE_VERSION] = self.service_version
+            if self.environment:
+                otel_resource_attributes[RESOURCE_ATTRIBUTES_DEPLOYMENT_ENVIRONMENT_NAME] = self.environment
```

Comment:
It's unfortunate that `ResourceAttributes.DEPLOYMENT_ENVIRONMENT` exists, but `ResourceAttributes.DEPLOYMENT_ENVIRONMENT_NAME` doesn't. And the first is deprecated...

https://opentelemetry.io/docs/specs/semconv/attributes-registry/deployment/

Reply:
Maybe open an otel issue? And put this link in code.